core/modules/filter/filter.pages.inc

```diff
@@ -126,7 +127,9 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
     '#title' => t('Alternative text'),
     '#type' => 'textfield',
     '#default_value' => isset($values['alt']) ? $values['alt'] : NULL,
+    '#description' => t('The alt attribute may be used by search engines, screen readers, and when the image cannot be loaded.'),
```

Comment:
I would love it if we could swap the order of `search engines, screen readers,` to `screen readers, search engines,`

Reply:
```suggestion
    '#description' => t('The alt attribute may be used by screen readers, search engines, and when the image cannot be loaded.'),
```